src/Uno.UI/Microsoft/UI/Xaml/Controls/MenuBar/MenuBar.cs

```diff
@@ -51,5 +51,32 @@ protected override void OnApplyTemplate()
 		}
 
 		internal bool IsFlyoutOpen { get; set; }
+
+		internal void OnMenuBarItemFlyoutPresenterPointerMoved(PointerRoutedEventArgs e)
```

Comment:
Can you add a comment that this is a workaround for lack of support of `OverlayInputPassThroughElement`? So that once `OverlayInputPassThroughElement` is properly implemented, we can properly implement this.

Reply:
I suppose it would. We would still need to add the logic for opening and closing the flyouts, but it will be in the PointerEntered and PointerExited of each menu item instead of PointerMoved of the flyout.

Reply:
@ramezgerges The logic for that is already in place

https://github.com/unoplatform/uno/blob/6afedadc731f8fd6efe44d27470e39bd7711ee54/src/Uno.UI/Microsoft/UI/Xaml/Controls/MenuBar/MenuBarItem.cs#L165-L174